{"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smiformresponseerrortypetypeinternalservererror"]}],"metadata":{"fragments":[{"kind":"identifier","text":"SMIFormResponseErrorTypeTypeInternalServerError"}],"roleHeading":"Global Variable","externalID":"c:@SMIFormResponseErrorTypeTypeInternalServerError","navigatorTitle":[{"kind":"identifier","text":"SMIFormResponseErrorTypeTypeInternalServerError"}],"symbolKind":"var","modules":[{"name":"SMIClientCore"}],"title":"SMIFormResponseErrorTypeTypeInternalServerError","role":"symbol"},"identifier":{"interfaceLanguage":"occ","url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorTypeTypeInternalServerError"},"sections":[],"kind":"symbol","schemaVersion":{"minor":3,"major":0,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore"]]},"abstract":[{"text":"An internal service error was triggered preventing the submission of the form.","type":"text"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["occ"],"tokens":[{"kind":"keyword","text":"extern"},{"kind":"text","text":" "},{"kind":"typeIdentifier","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorType","text":"SMIFormResponseErrorType","preciseIdentifier":"c:SMIFormResponseErrorType.h@T@SMIFormResponseErrorType"},{"kind":"text","text":" "},{"kind":"keyword","text":"const"},{"kind":"text","text":" "},{"kind":"identifier","text":"SMIFormResponseErrorTypeTypeInternalServerError"},{"kind":"text","text":";"}],"platforms":["iOS"]}]}],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIFormResponseErrorType":{"type":"topic","abstract":[{"type":"text","text":"An enum that defines "},{"type":"codeVoice","code":"SMIFormResponseErrorType"},{"type":"text","text":" errors returned due to a failed form submission."},{"type":"text","text":" "},{"type":"text","text":"Possible values:"}],"fragments":[{"kind":"identifier","text":"SMIFormResponseErrorType"}],"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorType","title":"SMIFormResponseErrorType","kind":"symbol","role":"symbol","url":"\/documentation\/smiclientcore\/smiformresponseerrortype"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIFormResponseErrorTypeTypeInternalServerError":{"kind":"symbol","url":"\/documentation\/smiclientcore\/smiformresponseerrortypetypeinternalservererror","title":"SMIFormResponseErrorTypeTypeInternalServerError","type":"topic","abstract":[{"type":"text","text":"An internal service error was triggered preventing the submission of the form."}],"role":"symbol","fragments":[{"kind":"identifier","text":"SMIFormResponseErrorTypeTypeInternalServerError"}],"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIFormResponseErrorTypeTypeInternalServerError"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"url":"\/documentation\/smiclientcore","kind":"symbol","type":"topic","role":"collection","abstract":[],"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","title":"SMIClientCore"}}}